closed/OpenJ9.gmk

```diff
@@ -63,11 +63,14 @@ else
   OMR_EXTRA_CONFIGURE_ARGS := "--enable-warnings-as-errors OMR_TREAT_WARNINGS_AS_ERRORS=1"
 endif
 
-# disable mingw on windows, use default c compiler
 ifeq ($(OPENJDK_TARGET_OS),windows)
+  # disable mingw on windows, use default c compiler
   EXPORT_NO_USE_MINGW := NO_USE_MINGW=true
+  # set the output directory for shared libraries
+  OPENJ9_LIBS_OUTPUT_DIR := bin
 else
   EXPORT_NO_USE_MINGW :=
+  OPENJ9_LIBS_OUTPUT_DIR := lib
```

Comment:
Can you explore whether it makes sense to use `LIBS_TARGET_SUBDIR` via `make/Images.gmk` rather than reinvent it as `OPENJ9_LIBS_OUTPUT_DIR`?

Reply:
LIBS_TARGET_SUBDIR is a local make variable defined in Images.gmk (not in spec.gmk). It is not inherited. 
An extension mechanism seems to be available for images too via Images-pre.gmk and Images-post.gmk. We could take advantage of it when implementing issue #21. That would remove  some of the code from OpenJ9.gmk to the image extensions (e.g. the stage build jdk, jdk/jre image, copy j9 shared libs).

Reply:
Yes, I also hope that when we get to the bottom of #21 openj9 will fit more cleanly into the openjdk build model.